framework/tlvf/yaml/tlvf/WSC/eWscLengths.yaml

```diff
@@ -21,4 +21,5 @@ eWscLengths:
   WSC_MAX_NETWORK_KEY_LENGTH: 64
   WSC_PUBLIC_KEY_LENGTH: 192
   WSC_KEY_WRAP_AUTH_LENGTH: 8
+  WSC_AUTHENTICATOR_LENGTH: 8
```

Comment:
add final newline

Reply:
There is a newline. If there isn't, it would show as red.